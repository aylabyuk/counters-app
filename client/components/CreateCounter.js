import React from "react";
import { Button, Input } from "antd";

const AddCounter = () => {
  return (
    <div style={{ display: "flex", width: "100%" }}>
      <Input size="default" placeholder="New Counter" />
      <Button
        style={{ marginLeft: "5px" }}
        size="default"
        type="primary"
        icon="plus"
      >
        Add
      </Button>
    </div>
  );
};

export default AddCounter;
