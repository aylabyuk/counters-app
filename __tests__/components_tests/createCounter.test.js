import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import to_json from "enzyme-to-json";

import CreateCounter from "../../client/components/CreateCounter";

configure({ adapter: new Adapter() });

describe("<CreateCounter />", () => {
  it("Renders correctly", () => {
    const wrapper = shallow(<CreateCounter />);
    expect(to_json(wrapper)).toMatchSnapshot();
  });
});
