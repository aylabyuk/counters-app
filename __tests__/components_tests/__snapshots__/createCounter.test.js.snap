// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateCounter /> Renders correctly 1`] = `
<Form
  colon={true}
  hideRequiredMark={false}
  layout="horizontal"
  onSubmit={[Function]}
  style={
    Object {
      "display": "flex",
      "width": "100%",
    }
  }
>
  <Input
    disabled={false}
    onChange={[Function]}
    placeholder="New Counter"
    size="default"
    type="text"
    value=""
  />
  <Button
    block={false}
    disabled={true}
    ghost={false}
    htmlType="submit"
    icon="plus"
    loading={false}
    size="default"
    style={
      Object {
        "marginLeft": "5px",
      }
    }
    type="primary"
  >
    Add
  </Button>
</Form>
`;
